README.md

```diff
@@ -22,59 +22,34 @@ export PATH=$PATH:/path/to/Qt/5.14.2/gcc_64/bin
 export PATH=$PATH:/path/to/Qt/5.14.2/clang_64/bin
 ```
 
-### 3. Clone and build DOtherside
-
-For Linux:
-
-```
-sudo apt-get install build-essential libgl1-mesa-dev
-sudo apt-get install doxygen
-```
-
-```
-git clone https://github.com/filcuc/DOtherSide
-cd DOtherSide
-mkdir build && cd build
-cmake ..
-make
-```
-
-### 4. Setup Library Path
-
+### 3. Clone the repo
 ```
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/path/to/dotherside/build/lib
+git clone https://github.com/status-im/nim-status-client/ --recurse-submodules
```

Comment:
The need to fetch/update the submodules while cloning the project was mitigated by the special default target in the Makefile that does that for you the first time you run `make` with the submodules missing.